docs/hbase.md

```diff
@@ -0,0 +1,102 @@
+# `hbase` Metrics Receiver
+
+The `hbase` metric receiver can fetch stats from a HBase server's Java Virtual Machine (JVM) via [JMX](https://www.oracle.com/java/technologies/javase/javamanagement.html). It collects metrics specific to the local region server, as well as metrics presented by the Master node if the node being monitored is indeed the Master.
+
+For High Availability configurations it is recommended to monitor the active Master node only. Monitoring the active Master and its replicants will result in duplicate metrics for the Master node.
```

Comment:
Please try to make this paragraph clearer to explain that it only applies to master nodes; I read it as saying "you should only configure the receiver on one node", which isn't true - you should configure it on "one master node and  all region server nodes".

If the "if the node being monitored is indeed the Master" is accurate, can't you just configure it on all of the nodes and there just won't be master metrics on the replicants until the replicant becomes the master?

Reply:
Hbase has some pending discussions going on in the `Category 3 - Regional metrics and master metrics` section of the `Duplicate / Overlapping Metrics Issue from Clustered Applications` doc.

The phrasing here might be adjusted based on the decision. We'll need to resolve that conversion before releasing this.

Reply:
Updated